README.md

```diff
@@ -1 +1,87 @@
-# knowledge-agents
\ No newline at end of file
+<!--
+ * Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+# Tractus-X Knowledge Agents Reference Implementations (KA-RI)
+
+![GitHub contributors](https://img.shields.io/github/contributors/eclipse-tractusx/knowledge-agents)
+![GitHub Org's stars](https://img.shields.io/github/stars/catenax-ng)
+![GitHub](https://img.shields.io/github/license/eclipse-tractusx/knowledge-agents)
+![GitHub all releases](https://img.shields.io/github/downloads/eclipse-tractusx/knowledge-agents/total)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=eclipse-tractusx_knowledge-agents&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=eclipse-tractusx_knowledge-agents)
+
+Tractus-X Knowledge Agents Reference Implementations (KA-RI) is a product of the [Catena-X Knowledge Agents Kit (about to move to: Tractus-X Knowledge Agents Kit)](https://catenax-ng.github.io/product-knowledge) implementing the "binding" modules of the CX-0084 standard (Federated Queries in Dataspaces).
+
```

Comment:
Do not link to catenax-ng repositories/GH Org.

Reply:
choose a neutral tractusx url alias that can be moved/or replaced when the Kit is released